### PR TITLE
Clean up unused code

### PR DESCRIPTION
**Pull request by ericholguin**
In the interest of reducing the amount of code we maintain this PR removes files that are not in use or no longer apply to the project.

- Removed components in `infrastructre/docker/`, not in use, cdn-in-a-box performs the same functionality
- Removed `misc/jira_github_issue_import.py`, the project does not use JIRA
- Removed `traffic_ops/install/bin/install_go.sh`, `traffic_ops/install/bin/todb_bootstrap.sh` and `traffic_ops/install/bin/install_goose.sh` are no longer in use

<hr/>

## PR submission checklist
- [ ] This PR has tests 
- [ ] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -66,6 +66,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 - [#7204](https://github.com/apache/trafficcontrol/pull/7204) *Traffic Control Cache Config (t3c)* strategies.yaml hash_key only for consistent_hash
 
+### Removed
+- Remove components in `infrastructre/docker/`, not in use, cdn-in-a-box performs the same functionality
+- Remove `misc/jira_github_issue_import.py`, the project does not use JIRA
+- Remove `traffic_ops/install/bin/_postinstall.pl` and `traffic_ops/install/lib/`, _postinstall.py is the recommended script to use for install
+- Remove `traffic_ops/install/bin/convert_profile/`, this script is outdated and is for use on an EOL ATS version
+- Remove `traffic_ops/install/bin/install_go.sh`, `traffic_ops/install/bin/todb_bootstrap.sh` and `traffic_ops/install/bin/install_goose.sh` are no longer in use
+- 
+
```
Comment (ocket8888) [inline CHANGELOG.md:76]: These need a PR or issue link, and you've got an extra blank list item at the end.

**Review by ocket8888 — CHANGES_REQUESTED** [review]
Did you consult with Evan Zelkowitz about the removal of `convert_profile`?

File: traffic_ops/install/bin/_postinstall.pl [hunk 2]
```diff
@@ -1,919 +0,0 @@
-#!/usr/bin/perl
```
Comment (zrhoffman) [inline traffic_ops/install/bin/_postinstall.pl:1]: While it's true that the `master` branch does not use `_postinstall.pl`, that does not mean that `_postinstall.pl` itself is unused. In ATC 6, it was deprecated, pending removal. So it should be removed in its own PR, rather than in a PR of things that are totally unused.

**Review by zrhoffman — CHANGES_REQUESTED** [review]

**Comment by ericholguin** [comment]
> Did you consult with Evan Zelkowitz about the removal of `convert_profile`?

Yes, he said we have not been using it, however maybe @smalenfant may have a different opinion on its removal?

**Review by zrhoffman — CHANGES_REQUESTED** [review]
The TO RPM fails to build:

```
+ cd install/bin/convert_profile
traffic_ops/build/build_rpm.sh: line 94: cd: install/bin/convert_profile: No such file or directory
```

**Comment by smalenfant** [comment]
> > Did you consult with Evan Zelkowitz about the removal of `convert_profile`?
> 
> Yes, he said we have not been using it, however maybe @smalenfant may have a different opinion on its removal?

I do use this convert_profile when I upgrade ATS. I have those rules still in use on our local repository.
Can we expand on "outdated" ?

```
convert_ats7_to_ats8.yaml
convert_ats8_to_ats9.yaml
```

**Comment by ericholguin** [comment]
> I do use this convert_profile when I upgrade ATS. I have those rules still in use on our local repository. Can we expand on "outdated" ?
> 
> ```
> convert_ats7_to_ats8.yaml
> convert_ats8_to_ats9.yaml
> ```

The files removed from the `convert_profile` directory were:
- convert622to713.json
- convert622to713.yaml 
- convert_profile.go 
- convert_profile_test.go

Outdated here being the ATS versions those scripts reference: 6.2.2 and 7.1.3. If these are still in use I can re-add them to this PR.

**Review by zrhoffman — CHANGES_REQUESTED** [review]
Building the TO RPM fails:

```shell
+ sed -i.bak s/__GO_VERSION__/1.19.8/g install/bin/install_go.sh
sed: can't read install/bin/install_go.sh: No such file or directory
```

**Comment by ericholguin** [comment]
> Building the TO RPM fails:
> 
> ```shell
> + sed -i.bak s/__GO_VERSION__/1.19.8/g install/bin/install_go.sh
> sed: can't read install/bin/install_go.sh: No such file or directory
> ```

traffic_ops.spec uses a variable from that script: `GO_TARBALL_VERSION=__GO_VERSION__`
Should this file be brought back or should that command be replaced?

**Comment by zrhoffman** [comment]
> > Building the TO RPM fails:
> > ```shell
> > + sed -i.bak s/__GO_VERSION__/1.19.8/g install/bin/install_go.sh
> > sed: can't read install/bin/install_go.sh: No such file or directory
> > ```
> 
> traffic_ops.spec uses a variable from that script: `GO_TARBALL_VERSION=__GO_VERSION__` Should this file be brought back or should that command be replaced?

That isn't needed anymore because Go is no longer installed because we no longer use Goose, which is why install/bin/install_go.sh was removed

**Review by zrhoffman — APPROVED** [review]
Looks good!
